src/main/java/com/werfen/report/service/excel/ExcelGridReportService.java

```diff
@@ -1,30 +1,27 @@
 package com.werfen.report.service.excel;
 
 import com.werfen.report.model.*;
-import org.apache.commons.io.IOUtils;
+import com.werfen.report.service.excel.template.DataSheetBuilder;
+import com.werfen.report.service.excel.template.SummarySheetBuilder;
 import org.apache.poi.ss.usermodel.*;
-import org.apache.poi.xssf.usermodel.XSSFFont;
 import org.apache.poi.xssf.usermodel.XSSFWorkbook;
 
-import java.io.FileInputStream;
 import java.io.FileOutputStream;
 import java.io.IOException;
-import java.util.List;
-import java.util.Optional;
 import java.util.logging.Logger;
 
-import static java.util.Objects.nonNull;
-
 public class ExcelGridReportService {
 
-    private static final int BASE_COLUMN_WIDTH = 256 * 7;
     Logger log = Logger.getLogger(ExcelGridReportService.class.getName());
 
     public void export(String filePath, GridReportConfiguration gridReportConfiguration, GridPageDataSource dataSource) throws IOException {
         Workbook workbook = new XSSFWorkbook();
 
-        addDataSheet(workbook, dataSource, gridReportConfiguration.getGridColumnConfigurations());
-        addSummarySheet(workbook, gridReportConfiguration);
+        DataSheetBuilder dataTemplate=new DataSheetBuilder();
```

Comment:
I am OK on splitting the generation of excel sheets in separated classes. However, naming these new classes as "builders" doesn't make me feel comfortable at all. The build() method returns a void and that can be confusing. I would expected a builder to do something and return it. Why not keeping the name of the classes as it is and renaming the build method as add()?